src/components/reservation/ReservationModalBody.tsx

```diff
@@ -128,12 +139,47 @@ export const ReservationModalBody = ({
     ? getFutureDateString(selectedInterest)
     : null;
 
+  const {
+    isReservable: isReservableOnAnotherLibrar,
+    pids: pidsOnAnotherLibrar
+  } = getReservableOnAnotherLibrary(manifestationsToReserve);
+
   const saveReservation = () => {
     if (!manifestationsToReserve || manifestationsToReserve.length < 1) {
       return;
     }
+
+    if (isReservableOnAnotherLibrar && patron) {
+      const { patronId, address, name, emailAddress } = patron;
+
+      mutateOpenOrder(
+        {
+          input: {
+            pids: [...pidsOnAnotherLibrar],
+            pickUpBranch: selectedBranch
+              ? removePrefixFromBranchId(selectedBranch)
+              : "",
+            expires: selectedInterest?.toString() || "90",
```

Comment:
"90" should not be hard coded like this. We need it in our global config scope. Please take a look in core how it has been done. It is quite a mess there though, because ITK and Reload has been dealing with config in slightly different manners. We can talk about it... that is easier.